.gitignore

```diff
@@ -0,0 +1 @@
+.bin
```

Comment:
as you see, here github gives you a hing that 'last line' missing. (this red sign). 

In programming really good practice to have last line as an empty one. So add this empty line in the end of this file. 

In case of txt files there is no such requirement, but for files which related to your progreamm (in future this is files with `.java` `.yml` `.sql` and many other extentions it is required